modules/claims_api/app/services/claims_api/disability_compensation/docker_container_service.rb

```diff
@@ -10,21 +10,21 @@ class DockerContainerService < ServiceBase
       LOG_TAG = '526_v2_Docker_Container_service'
 
       def upload(claim_id)
-        log_job_progress(claim_id,
-                         'Docker container service started')
-
         auto_claim = get_claim(claim_id)
+
+        log_job_progress(claim_id, 'Docker container service started', auto_claim.transaction_id)
```

Comment:
@tycol7 I am not seeing the transaction_id
<img width="1178" alt="Screenshot 2024-08-29 at 8 40 29 AM" src="https://github.com/user-attachments/assets/e3396412-a92a-46b3-b967-4a12124a6653">